### PR TITLE
Release new dependency versions for commons-io, solr, bcpkix-jdk15on.

### PR DESCRIPTION
**Pull request by bhagyasakalanka**
Related issues wso2/product-is#12485
## Purpose
- bcpkix-jdk15on - 1.68.0.wso2v1
- coomons-io - 2.7.0.wso2v1
- solr - 5.5.5.wso2v2
 solr release is done to increase range of common-io.

### CODE REVIEW

File: bcpkix/1.68.0.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,132 @@
+<!--
+ ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (ashensw) [inline bcpkix/1.68.0.wso2v1/pom.xml:2]: Fix the license.

File: commons-io/2.7.0.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,87 @@
+<!--
+ ~ Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (ashensw) [inline commons-io/2.7.0.wso2v1/pom.xml:2]: Fix the license.

File: solr/5.5.5.wso2v2/pom.xml [hunk 3]
```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (ashensw) [inline solr/5.5.5.wso2v2/pom.xml:3]: Fix the license.

File: bcpkix/1.68.0.wso2v1/pom.xml (hunk 1 above)
Comment (bhagyasakalanka) [reply bcpkix/1.68.0.wso2v1/pom.xml]: fixed

File: solr/5.5.5.wso2v2/pom.xml (hunk 3 above)
Comment (bhagyasakalanka) [reply solr/5.5.5.wso2v2/pom.xml]: fixed

File: commons-io/2.7.0.wso2v1/pom.xml (hunk 2 above)
Comment (bhagyasakalanka) [reply commons-io/2.7.0.wso2v1/pom.xml]: fixed

**Review by sumedhe — APPROVED** [review]

File: bcpkix/1.68.0.wso2v1/pom.xml [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+    <dependencies>
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcpkix-jdk15on</artifactId>
+            <version>1.68</version>
+            <!-- By setting optional false, dependent projects of this orbit bundle will be able to see bcprov-jdk15on library as a transitive dependency.
+             This is required because we are embedding bcprov-jdk15on library inside this orbit bundle without extracting the content,
+             becuase bcprov-jdk15on library is signed.
+             If you embed a dependency and set optional to true, then dependent projects will not be alble to compile their source with this orbit bundle.-->
```
Comment (sumedhe) [inline bcpkix/1.68.0.wso2v1/pom.xml:42]: Lines seems contain more than 120 char. Shall we split them?

Comment (bhagyasakalanka) [reply bcpkix/1.68.0.wso2v1/pom.xml]: Fixed

**Pull request merged**